src/classes/RD_RecurringDonations_TEST.cls

```diff
@@ -309,7 +309,9 @@ public class RD_RecurringDonations_TEST {
             WHERE Opportunity.npe03__Recurring_Donation__c = :r1.id
         ];
         system.assertEquals(2,installmentCRs.size());
-        system.assertEquals(system.label.npe03.RecurringDonationContactRole,installmentCRs[0].Role);
+        if (!OPP_AutomatedSoftCreditsService.isOrganizationalAccount(c.Account.npe01__SYSTEMIsIndividual__c)) {
```

Comment:
this test gets called three times, once with 1:1, once with HH Account, and once with Individual Bucket accounts.  So in all three cases, these are not Organizational accounts.  If we think this routine might get called with an org account, then I'd like to see an else assert checking for the other value.  If that isn't going to happen, then I'd rather just see an assert that these aren't org accounts, rather than have an if with no else.  make sense?

Reply:
The issue that was present earlier development is no longer surfacing. I have removed the if check.